src/proxyManagerConfiguration.js

```diff
@@ -1,13 +1,31 @@
 import vtkProxySource from 'vtk.js/Sources/Proxy/Core/SourceProxy';
 import vtkGeometryRepresentationProxy from 'vtk.js/Sources/Proxy/Representations/GeometryRepresentationProxy';
-import vtkMoleculeRepresentationProxy from 'vtk.js/Sources/Proxy/Representations/MoleculeRepresentationProxy';
 import vtkVolumeRepresentationProxy from 'vtk.js/Sources/Proxy/Representations/VolumeRepresentationProxy';
 import vtkSliceRepresentationProxy from 'vtk.js/Sources/Proxy/Representations/SliceRepresentationProxy';
 import vtkPiecewiseFunctionProxy from 'vtk.js/Sources/Proxy/Core/PiecewiseFunctionProxy';
 import vtkLookupTableProxy from 'vtk.js/Sources/Proxy/Core/LookupTableProxy';
 
 import ItkVtkView from './ItkVtkViewProxy';
 
+const commonInteractor = [
+  { type: 'pan', options: { button: 3 } }, // Pan on Right button drag
+  { type: 'pan', options: { button: 1, shift: true } }, // Pan on Shift + Left button drag
+  { type: 'zoom', options: { button: 1, control: true } }, // Zoom on Ctrl + Left button drag
+  { type: 'zoom', options: { dragEnabled: false, scrollEnabled: true } }, // Zoom on scroll
+];
+
+const interactorStyle3D = [
+  Array.from(commonInteractor).concat([
```

Comment:
Concat create a new array, so you don't need `Array.from`.

Reply:
:+1: I'll remove `Array.from`.